core/services/functions/request.go

```diff
@@ -1,10 +1,19 @@
 package functions
 
 type RequestData struct {
```

Comment:
The CBOR payload from on-chain has secrets encoded as bytes, but the adapter expects the secrets to be base64 string.  This is resolved by adding the new AdapterRequestData struct

Reply:
Wait, that means the current RequestData is incorrect, right? I'll change it to bytes and add a test for that field, my bad!

Reply:
As soon as that PR is merged, I'll rebase and remove the additional struct